crates/holochain/src/core/sys_validate/tests.rs

```diff
@@ -402,26 +402,11 @@ async fn check_app_entry_def_test() {
     // ## Can get the entry from the entry def
     let app_entry_def_5 = AppEntryDef::new(0.into(), 0.into(), EntryVisibility::Public);
     assert_matches!(
-        check_app_entry_def(&dna_hash, &app_entry_def_5, &conductor_handle).await,
+        check_app_entry_def(&app_entry_def_5, &dna_hash, &conductor_handle).await,
         Ok(_)
     );
 }
 
-#[tokio::test(flavor = "multi_thread")]
```

Comment:
Did you not update this test because `check_entry_not_private` is lexically simple enough to confirm?

Reply:
I'm not sure why I removed this. Probably because the function changed significantly. But it really should be replaced with another test. I'll write a new one.

Reply:
Ah yes. It's because I removed the function altogether!

Reply:
And the replacement function is very straighforward, lexically